.devcontainer/scripts/runItOnGo.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+FOLDER_PATH=$1
+if [ -z "${FOLDER_PATH}" ]; then
+    FOLDER_PATH=$PWD
+fi
+
+BASE_PATH=$2
+if [ -z "${BASE_PATH}" ]; then
+    BASE_PATH=$WORKSPACE_FOLDER
+fi
+
+if [ "$FOLDER_PATH" = "$BASE_PATH" ]; then
+    echo "Skipping lcov creation for base path"
+    exit 1
+fi
+
+FOLDER_NAME=${FOLDER_PATH#"$BASE_PATH/"}
+FILE_NAME="$(echo "$FOLDER_NAME" | sed 's/\//-/g').ginkgo"
+FILE_PATH="/tmp/$FILE_NAME"
+
+
+echo "-- Start $FOLDER_NAME ($(date '+%T')) --"
+
+TIMEFORMAT=' - Ginkgo tests finished in: %R seconds'
+time ginkgo --label-filter="!e2e" --keep-going --timeout=5m --output-dir=/tmp --coverprofile="$FILE_NAME" --covermode=atomic --cover -r -p "$FOLDER_PATH" || true
```

Comment:
I have a local change to the `Makefile` to set `output-dir` to the equivalent of `${WORKSPACE_FOLDER}/coverage` and just ignore that whole folder in `.gitignore`. Maybe we can do that to avoid putting files in every dir?  
That also seems to match what the `markis.code-coverage` plugin expects as per its [README](https://github.com/markis/vscode-code-coverage#getting-started), so we wouldn't need a custom setting.

Reply:
Seems like I lost track over all the things I tried to get this features performance up. 😅

I'll change it. 👍

Reply:
I moved all lcov files in a coverage folder but had to change the default search pattern since the default one behaved flaky.
All container specifc settings are now in the devcontainer.json.
All generic ones in the settings.json.